_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/provision-connector.adoc

```diff
@@ -1,12 +1,12 @@
 You can choose one of two options for provisioning a fully managed Kafka Connect Datagen source connector in Confluent Cloud.  Either option will use the connector configuration file `datagen-source-config.json` that you created in the previous step.
 
-*Option 1.* You can use the Confluent Cloud CLI which provides the `ccloud connector create` command allowing you to pass in the configuration file from the previous step.
+*Option 1.* You can use the Confluent CLI which provides the `ccloud connector create` command allowing you to pass in the configuration file from the previous step.
```

Comment:
@bbejeck Looks like `ccloud` instance here needs to be updated.

Actually just did a `grep` and seems like a few others too?

```
_includes/tutorial-content.html:   {% if site.data.harnesses[page.static_data][page.stack].ccloud %}
_includes/tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/delete-connector.sh:ccloud connector delete <connector id>
_includes/tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/create-cloud-api-key.sh:ccloud api-key create --resource cloud -o json > cloud-api-key.json
_includes/tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/check-connector.sh:ccloud connector list
_includes/tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/list-connector.sh:ccloud connector list
_includes/tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/create-connector.sh:ccloud connector create --config datagen-source-config.json 
_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/get-ccloud.adoc:Install the link:https://docs.confluent.io/current/cloud/cli/install.html[Confluent CLI] and login with the command ``ccloud login --save``, using your Confluent Cloud username and password.
_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/provision-connector.adoc:*Option 1.* You can use the Confluent CLI which provides the `ccloud connector create` command allowing you to pass in the configuration file from the previous step.
_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/provision-connector.adoc:- Confluent Cloud environment id into the configuration parameter `ENVIRONMENT` (use the command `ccloud environment list` to view the active environment).
```

Reply:
I'm not sure about changing `_includes/tutorial-content.html:   {% if site.data.harnesses[page.static_data][page.stack].ccloud %}` I think the usage here is internal for rendering instructions

Reply:
@ybyzek there's a new PR https://github.com/confluentinc/kafka-tutorials/pull/987 addressing these changes